<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntVec3Widget</class>
 <widget class="QWidget" name="IntVec3Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>37</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="IntVec3WidgetLabel">
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="xData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="yData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="zData"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xData</tabstop>
  <tabstop>yData</tabstop>
  <tabstop>zData</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
